src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -169,6 +167,17 @@ private void buildElement(MessageMLParser context, org.w3c.dom.Element element)
       }
 
       addChild(child);
+    } else if(element.getNodeName().equals(Div.MESSAGEML_TAG)) {
+      /*
+      When converting from PresentationML -> MessageML some elements are not converted
```

Comment:
In reality the conversion it is not PresentationML to MessageML, but to **MessageML tree object** (that can be in PresentationML or MessageML format)